app/IATI/Services/Audit/AuditService.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace App\IATI\Services\Audit;
+
+use App\IATI\Repositories\Audit\AuditRepository;
+use Exception;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+
+class AuditService
+{
+    protected AuditRepository $auditRepository;
+
+    public function __construct(AuditRepository $auditRepository)
+    {
+        $this->auditRepository = $auditRepository;
+    }
+
+    /**
+     * @param array|object $auditables
+     * @param Request $request
+     * @param string $event
+     * @param string $eventFormat
+     * @param bool $auditMultiple
+     *
+     * @return void
+     */
+    public function audit_event(array|object $auditables, Request $request, string $event, string $eventFormat = '', bool $auditMultiple = false): void
```

Comment:
write variables and method name in camelcase and add method description in phpdocs